### ISSUE TITLE
AttributeError: function 'SharedMemoryControlContextCudaRegister' not found (Windows 10)

### ISSUE THREAD

**Issue by JulianOrteil** [NVIDIA/triton-inference-server#1458]
**Description**
After successfully building the trtis-clients and installing the wheel file into python, I am confronted with this error:
![image](https://user-images.githubusercontent.com/23083317/81600659-981f0e80-938f-11ea-8aa9-b4d71f2a98ab.png)

Environment information:
- OS: Windows 10 Version 10.0.19041 Build 19041
- Python: 3.7.7
- Path includes the 'vcpkg\installed\x64-windows\bin' for everything installed using vcpkg in the guide below; otherwise I get a "OSError: [WinError 126] The specified module could not be found"
- tensorrtserver version (as listed by pip): 1.13.0.dev0

**Triton Information**
20.03

Are you using the Triton container or did you build it yourself?
Built myself (clients only)

**To Reproduce**
- Build the clients following the guide [here](https://github.com/NVIDIA/triton-inference-server/blob/master/docs/client.rst#windows-10).
- Then run **pip install tensorrtserver-1.13.0.dev0-py3-none-win_amd64.whl** from the build/trtis-clients/install/python directory.
- Then, in a command prompt, run:
```
> python
>>> import tensorrtserver.api
```

Describe the models (framework, inputs, outputs), ideally include the model configuration file (if using an ensemble include the model configuration file for that as well).
N/A

**Expected behavior**
Successful import of tensorrtserver.

**Comment by deadeyegoodwin** [comment]
Sorry that the windows client is so broken. We will improve our test coverage. Can you try applying #1461 before building to see if it fixes your problem.

**Comment by JulianOrteil** [comment]
@deadeyegoodwin 

You're good. I completely understand these issues. I'm not bothered at all.
Would I be correct in assuming you are porting from Linux to Windows instead of rebuilding from scratch?

**Comment by JulianOrteil** [comment]
And thank you for the timely fixes, I appreciate it. I'll let you know if I encounter further issues.

**Comment by deadeyegoodwin** [comment]
The windows version of the python client was done after the linux version, and both versions use a C/C++ shared library for some functionality. With the upcoming version 2 of the clients (can read about it in the Roadmap section of the README) the python client should be much easier to build/use on different platforms because it will be pure python (with an exception for shared-memory support).

**Comment by JulianOrteil** [comment]
I was able to successfully build, install and run the clients with the changes made in #1461. I got an error message from image_client.py (Exception: expecting 1 output, got 3) which is expected as I have an object detection model currently loaded on the server.
This tells me that everything up until sending the inference request is working. I'll give a better update when I am at my workstation tomorrow morning.

Thank you again for your hard work @deadeyegoodwin (and whoever helped with this issue)!
